server/services/featureFlagService.ts

```diff
@@ -0,0 +1,42 @@
+import { ClientTokenAuthentication, FliptClient } from '@flipt-io/flipt'
+import config from '../config'
+import logger from '../../logger'
+
+export default class FeatureFlagService {
+  fliptClient: FliptClient | null
+
+  namespaceKey: string | null
+
+  constructor() {
+    if (config.fliptEnabled) {
+      this.fliptClient = new FliptClient({
+        url: config.apis.flipt.url,
+        authenticationStrategy: new ClientTokenAuthentication(config.apis.flipt.token),
+      })
+      this.namespaceKey = config.apis.flipt.namespace
+    }
+  }
+
+  async getBooleanFlag(flag: string): Promise<boolean> {
```

Comment:
When we have flags in the service we will want to make sure this argument is strongly typed